AutomationHelpers/src/UserCodeCollections/PopupWatcherLibrary.cs

```diff
@@ -77,7 +77,7 @@ public static PopupWatcher PauseWhileExists(RepoItemInfo findElement)
             watcher.Watch(findElement, RequestPauseWhileExists);
             watcher.Start();
             watchers.Add(key, watcher);
-            Report.Info("Started watching for: " + findElement.AbsolutePath);
+            Report.Info(string.Format("Started watching for: '{0}'", findElement.FullName));

```

Comment:
There is still a superfluous ":" and a no "." at the end of the sentence

Reply:
Good catch! Thx for that.